.github/workflows/sdk-javascript-gate-check.yml

```
@@ -0,0 +1,16 @@
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+permissions:
+  contents: read
+  checks: read
+
+jobs:
+  javascript-sdk-gate:
+    uses: ./.github/workflows/utility-gated-requirement-checks.yml@feature/workflow-gate-pattern
```

actionlint (1.7.7)

reusable workflow call "./.github/workflows/utility-gated-requirement-checks.yml@feature/workflow-gate-pattern" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)